packages/edit-site/src/components/sidebar-navigation-screen-global-styles/index.js

```diff
@@ -53,9 +55,19 @@ export function SidebarNavigationItemGlobalStyles( props ) {
 		<SidebarNavigationItem
 			{ ...props }
 			onClick={ () => {
-				// switch to edit mode.
+				// Disable distraction free mode.
+				toggleFeature( 'distractionFree', false );
```

Comment:
Why would we need to always toggle this feature? Additionally, this function doesn't accept a second argument, so the `false` here does nothing.

I'm going to remove this as part of https://github.com/WordPress/gutenberg/issues/50924, but let me know if we want it for some reason and should just make it work properly.

Reply:
@draganescu can you give a quick testing [here](https://github.com/WordPress/gutenberg/pull/51964) that the DFM behavior is the wanted one, when we click to open styles? Thanks!

Reply:
Yes, accesing the edit styles link for themes that don't have styles disables DFM.